src/util/localStorage.js

```diff
@@ -0,0 +1,18 @@
+const TODOS = "todos"
+
+const getTodos = () => {
+    const value = localStorage.getItem(TODOS);
+
+    return value === null ? [] : JSON.parse(value);
```

Comment:
아래 코드처럼 null 병합 연산자인 ??를 사용해보면 좋을 것 같아요!
```suggestion
    return JSON.parse(value) ?? [];
```
최근에 저도 알게 된 연산자인데 어떤 값을 할당하고 싶은지에 집중할 수 있었어요. 

병합 연산자에 대한 설명은 [이 글](https://ko.javascript.info/nullish-coalescing-operator)에 잘 정리가 되어있어요 😃

Reply:
저도 덕분에 null 병합 연산자 처음 알게되었네요 !

Reply:
오 그렇군요! 바로 적용하겠습니다!